runtime/queries/gitcommit/highlights.scm

```diff
@@ -0,0 +1,13 @@
+(subject) @markup.heading
+(path) @string.special.path
+(branch) @string.special.symbol
+(commit) @constant
+(item) @markup.link.url
+
+(change kind: "new file" @string)
+(change kind: "renamed" @attribute)
+(change kind: "deleted" @keyword)
+(change kind: "modified" @variable.builtin)
```

Comment:
I put these scopes to make my screenshot look pretty. Should there be new captures for these or is these something existing that would fit better?

Reply:
We should probably add new scopes for this since any other way to describe them using existing scopes would be a kludge.

Reply:
Yeah I was thinking something like the new markup scopes or the `@error`/`@warning`/`@info`/`@hint` stuff so it always comes out to "new file" = green, deleted = red, etc.

Reply:
We had "diff.added"/removed/modified scopes on this PR: https://github.com/helix-editor/helix/pull/467

I'm OK with using the theme.toml colors from there too since they're picked from my colibri palette

Reply:
Alternatively I also liked diff.plus/minus/delta

Reply:
What do you think about something like diff.moved for renames? We could just bin renamed and modified together under diff.delta but I kinda like having them separate

Reply:
I'd make it diff.delta.moved or diff.modified.moved, that way it can be styled together by only defining the top level scope, or separately by using specific scopes

Reply:
How about `git.added`, `git.modified`, `git.deleted` and `git.renamed` (`vcs.` as a prefix if we want to be more general) ? But it might clash with scope names in the future, for example when we have an inline diff view and gutter gitsigns.

Reply:
I wouldn't mind a `vcs.*` prefix instead of `diff.*`. I think we might be able to reuse these scopes for inline diffs and gitsigns without too much headache though